tdesign/src/Layout/ContentHeader/ContentHeader.tsx

```diff
@@ -9,28 +10,32 @@ export interface IContentHeaderProps {
 }
 
 const ContentHeader = ({ children, extraStyle = {} }: IContentHeaderProps) => {
+  const theme = useTheme();
+
+  const contentHeaderBannerStyle: SxProps<Theme> = {
+    display: "flex",
+    flexWrap: "wrap",
+    marginBottom: "1rem",
+    justifyContent: "space-between",
+    alignContent: "flex-start",
+    alignItems: "flex-end",
+    h2: {
+      color: "flambeeDarkGray.light",
+      letterSpacing: "0.005em",
+      fontWeight: "600",
+      display: "inline",
+      margin: 0,
+    },
+    [theme.breakpoints.down("sm")]: {
+      flexDirection: "column",
+      alignItems: "flex-start !important",
```

Comment:
I'm curious - why do we need `!important` here?

Reply:
Looks like some other changes make it unnecessary now. I was originally having trouble to alignItems due to some other style that was being used. `!important` helped me bring it to the top of the styling hierarchy, but no longer needed